backend/Application/AppGenerator.php

```diff
@@ -130,18 +126,10 @@ public function getMissingCoordinatesForCompanies(): void
                     text: "Trying to fetch coordinates for {$companies[$csvRow]["name"]}."
                 );
 
-                try {
-                    $rawCoordinates = $this->geocoder->coordinatesFromAddress(addressObject: $address);
-                    $companies[$csvRow]["coordinates"] = $rawCoordinates;
-                    $requiresSave = true;
-                } catch (Exception $exception) {
-                    OutputWriter::newLineToConsole($exception->getMessage());
-                    OutputWriter::newLineToConsole(
-                        "Couldn't fetch coordinates for {$companies[$csvRow]["name"]}."
-                        . " Check address or insert coordinates manually."
-                        . " Skipping..."
-                    );
-                }
+                $requiresSave = $requiresSave | $this->geocoder->coordinatesFromAddress(
```

Comment:
Could you explain this logic? I think we don't need this part `$requiresSave |`

Reply:
It will not execute correctly if the last fetched value couldn't be retrieved, because `$requiresSave` will be set to `false`.

I want to save file if it was successful for any entry. It doesn't save before going through each entry to file, and I don't want to change that due to how `fputcsv` command behaves.

I now introduced little change to this statement, by using `||` operand and moving `$requiresSave` to the second place.  So now it has two benefits:
1. It reduced complexity in already busy area of the code.
2. It no longer checks `$requiresSave` value if function returned `true` due to `||` optimization.